GeoSpatial.sln

```diff
@@ -5,6 +5,8 @@ VisualStudioVersion = 15.0.26124.0
 MinimumVisualStudioVersion = 15.0.26124.0
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "pims.api", "backend\pims.api.csproj", "{16BC0468-78F6-4C91-87DA-7403C919E646}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "pims.api.test", "test\BackendApi.Test\pims.api.test.csproj", "{19A6829C-20D8-4F66-9889-55C553ACE2B2}"
```

Comment:
Perhaps rename folder to reflect the change to the API (i.e. /test/backend/api/pims.api.test.csproject)

Reply:
I was going to suggest we move the whole folder under backend. That would match the frontend. Also the tests should be as close as possible to the code they check